src/test/java/com/google/devtools/build/lib/bazel/repository/downloader/HttpDownloaderTest.java

```diff
@@ -646,7 +646,7 @@ public void downloadAndReadOneUrl_checksumMismatch() throws IOException {
   @Test
   public void download_contentLengthMismatch_propagateErrorIfNotRetry() throws Exception {
     Downloader downloader = mock(Downloader.class);
-    DownloadManager downloadManager = new DownloadManager(repositoryCache, downloader);
+    DownloadManager downloadManager = new DownloadManager(repositoryCache, downloader, null);
```

Comment:
As the parameter is not marked as nullable, it would be safer to just pass in a new instance here (and below).

Reply:
Thanks! I chose to pass in a mock, so that we never accidentally access the internet from these unit tests.